Assignment1/diya singhal

```diff
@@ -0,0 +1,160 @@
+
+PROBLEM 1 CODE
```

Comment:
Submit the code as two separate files. 

Atleast check if the code compiles, don't randomly copy-paste.